src/main/react/application/Application.tsx

```diff
@@ -38,84 +37,77 @@ import AssignmentReport from "../features/report/Assignment/AssignmentReport";
 import ContractOverview from "../features/report/ContractOverview/ContractOverview";
 import AssignmentOverview from "../features/report/AssignmentOverview/AssignmentOverview";
 import { LoginFeature } from "../features/login/LoginFeature";
-import {usePerson} from "../hooks/PersonHook";
 
 const theme = getTheme("light");
 
 export const Application = () => {
   const status = useLoginStatus();
   const errors = useError();
 
+  useEffect(() => {
+    if (status) {
+      UserAuthorityUtil.setAuthorities(status.authorities);
+    } else {
+      UserAuthorityUtil.setAuthorities([]);
+    }
+  }, [status]);
+
   if (status == null) {
     return <AlignedLoader/>;
   }
 
   return (
     <Router>
       <ThemeProvider theme={theme}>
-        { status.isLoggedIn ? <RenderAuthenticated status={status} /> : <RenderUnauthenticated />}
-          <ErrorStack ErrorList={errors}/>
+        {status.isLoggedIn ? (
+          <RenderAuthenticated />
+        ) : (
+          <RenderUnauthenticated />
+        )}
+        <ErrorStack ErrorList={errors} />
       </ThemeProvider>
     </Router>
   );
 };
 
-const RenderAuthenticated = ({status}) => {
-  const [ person, handlePerson ] = usePerson();
-  const [ user, handleUser ] = useUserMe();
-
-  useEffect(() => {
-    if (status) {
-      UserAuthorityUtil.setAuthorities(status.authorities);
-      handlePerson(status.personId);
-      handleUser(status.userId);
-    }
-  }, [status]);
-
```

Comment:
handlePerson, and handleUser aren't needed here, as the `status: BootstrapState` contains this information already. Setting the same value again won't make any difference.

Then, we can actually move the `useEffect` to the only caller of this component, `Application`.